OmniBLE/OmnipodCommon/Pod.swift

```diff
@@ -55,15 +55,28 @@ public struct Pod {
 
     // Supported basal rates
     // Eros minimum scheduled basal rate is 0.05 U/H while for Dash supports 0 U/H.
-    // XXX would to have this based on productID to be able to share this with Eros.
-    public static let supportedBasalRates: [Double] = (1...600).map { Double($0) / Double(pulsesPerUnit) }
+    // Would need to have this value based on productID to be able to share this file with Eros.
+    public static let supportedBasalRates: [Double] = (0...600).map { Double($0) / Double(pulsesPerUnit) }
+
+    // The internal basal rate used for non-Eros pods
+    // Would need to have this value based on productID to be able to share this file with Eros.
+    public static let zeroBasalRate: Double = nearZeroBasalRate
 
     // Maximum number of basal schedule entries supported
     public static let maximumBasalScheduleEntryCount: Int = 24
 
     // Minimum duration of a single basal schedule entry
     public static let minimumBasalScheduleEntryDuration = TimeInterval.minutes(30)
 
+    // Max time between pulses for basal and temp basal extra timing command
+    public static let maxTimeBetweenPulses = TimeInterval(hours: 5)
+
+    // Near zero basal rate used for non-Eros pods for zero scheduled basal rates and temp basals
+    public static let nearZeroBasalRate = 0.01
```

Comment:
I intended the Pod.swift to be more of a class to expose certain user facing constants, not internal protocol constants. Could this be kept more in the encoding part of the code?